src/main/java/com/glodblock/github/network/CPacketLevelTerminalCommands.java

```diff
@@ -134,34 +130,7 @@ public IMessage onMessage(CPacketLevelTerminalCommands message, MessageContext c
                             message.side,
                             originalGui);
                 }
-                case ENABLE -> {
-                    if (con.getTarget() instanceof IClickableInTerminal clickableInterface) {
-                        Util.DimensionalCoordSide intMsg = clickableInterface.getClickedInterface();
-                        TileEntity tile = DimensionManager.getWorld(intMsg.getDimension())
-                                .getTileEntity(intMsg.x, intMsg.y, intMsg.z);
-                    }
-                }
-                case DISABLE -> {
-                    if (con.getTarget() instanceof IClickableInTerminal clickableInterface) {
-                        Util.DimensionalCoordSide intMsg = clickableInterface.getClickedInterface();
-                        TileEntity tile = DimensionManager.getWorld(intMsg.getDimension())
-                                .getTileEntity(intMsg.x, intMsg.y, intMsg.z);
-                    }
-                }
-                case ENABLE_ALL -> {
-                    if (con.getTarget() instanceof IClickableInTerminal clickableInterface) {
-                        Util.DimensionalCoordSide intMsg = clickableInterface.getClickedInterface();
-                        TileEntity tile = DimensionManager.getWorld(intMsg.getDimension())
-                                .getTileEntity(intMsg.x, intMsg.y, intMsg.z);
-                    }
-                }
-                case DISABLE_ALL -> {
-                    if (con.getTarget() instanceof IClickableInTerminal clickableInterface) {
-                        Util.DimensionalCoordSide intMsg = clickableInterface.getClickedInterface();
-                        TileEntity tile = DimensionManager.getWorld(intMsg.getDimension())
-                                .getTileEntity(intMsg.x, intMsg.y, intMsg.z);
-                    }
-                }
+                default -> {}
```

Comment:
Is removing these above cases with no substitute intentional?

Reply:
If I'm not mistaken, these cases don't do anything.

Reply:
Hm, I checked too and yeah seems these were unused